diff --git a/node_modules/fetch-h2/dist/lib/fetch-http1.js b/node_modules/fetch-h2/dist/lib/fetch-http1.js
index de1c4a5..07b1bdd 100644
--- a/node_modules/fetch-h2/dist/lib/fetch-http1.js
+++ b/node_modules/fetch-h2/dist/lib/fetch-http1.js
@@ -114,7 +114,12 @@ async function fetchImpl(session, input, init = {}, extra) {
                     }));
                 }
                 else {
-                    resolve(session.newFetch(request.clone(locationUrl), init, {
+                    // Remove auth from request. If the host is not the same as
+                    // the original request, we should not use the same auth.
+                    const newRequest = request.clone(locationUrl);
+                    newRequest.headers.delete("Authorization");
+                    newRequest.credentials = "omit";
+                    resolve(session.newFetch(newRequest, init, {
                         timeoutAt,
                         redirected: redirected.concat(url),
                     }));
